app/models/goal.py

```diff
@@ -1,5 +1,27 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import Integer, String
 from ..db import db
 
 class Goal(db.Model):
-    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    __tablename__ = "goals"
+
+    id: Mapped[int]    = mapped_column(Integer, primary_key=True, autoincrement=True)
```

Comment:
Nitpick: you have extra whitespaces here. Before you open a PR for code review at internship be sure that your code is cleaned up. Code that doesn't follow convention won't be reviewed or allowed to be merged because the team won't want to introduce inconsistencies into a codebase because it makes it more difficult to maintain and read. 

```suggestion
    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
```